File: uri/python/1020.py
# -*- coding: utf-8 -*-

idade_dias = int(input())

anos = 0
meses = 0
dias = 0

while idade_dias > 0:
    if idade_dias >= 365:
        anos = anos + 1
        idade_dias = idade_dias - 365
    elif idade_dias >= 30:
        meses = meses + 1
        idade_dias = idade_dias - 30
    else:
        dias = idade_dias
        break

print('{} ano(s)'.format(anos))
print('{} mes(es)'.format(meses))
print('{} dia(s)'.format(dias))

File: uri/C/beginner/1020.c
#include <stdio.h>


int calcularAno (int idadeDias){
	
	int ano=0;

	if(idadeDias >= 365){
		ano = idadeDias/365;
		
	}	

	return ano;
}

int calcularMes (int idadeDias){
	int mes=0;

	if(idadeDias >= 30){
		mes = idadeDias/30;
		
	}

	return mes;

}

int calcularDia (int idadeDias){

	int dia=0;

	if(idadeDias < 30){
		dia = idadeDias;
	}

	return dia;

}

int main ()
{

	int idadeDias;
	scanf("%d", &idadeDias);

	printf("%d ano(s)\n", calcularAno(idadeDias));
	idadeDias = idadeDias % 365; 
	printf("%d mes(es)\n", calcularMes(idadeDias));
	idadeDias = idadeDias % 30;
	printf("%d dia(s)\n", calcularDia(idadeDias));

	return 0;
}
